numba/cuda/tests/cudapy/test_enums.py

```diff
@@ -0,0 +1,116 @@
+"""
+Test cases adapted from numba/tests/test_enums.py
+"""
+
+import numpy as np
+
+from numba import int8, int16, int32
+from numba import cuda, vectorize
+from numba.tests.support import TestCase
+from numba.tests.enum_usecases import (
+    Color,
+    Shape,
+    Planet,
+    RequestError,
+    IntEnumWithNegatives
+)
+
+
+class EnumTest(TestCase):
```

Comment:
```suggestion
class EnumTest(CUDATestCase):
```

Reply:
This needs to be a `CUDATestCase` to ensure that the tests run in the right process - otherwise we get the "CUDA initialized before forking" error we see in GPU CI.